deploy-manage/deploy.md

```diff
@@ -41,22 +38,27 @@ Learn more about the [{{stack}}](/get-started/the-stack.md) to understand the co
 
 #### Managed by Elastic
 
-If you want to focus on using Elastic products rather than managing infrastructure, choose:
+If you want to focus on using Elastic products rather than managing infrastructure, choose one of the following options:
 
-- **Serverless**: Zero operational overhead, automatic scaling and updates, latest features
-- **Cloud hosted**: Balance of control and managed operations, choice of resources and regions
+- **{{serverless-full}}**: Zero operational overhead, automatic scaling and updates, latest features
+- **{{ech}}**: Balance of control and managed operations, choice of resources and regions
+
+Both of these options use [{{ecloud}}](/deploy-manage/deploy/elastic-cloud.md) as an orchestrator.
```

Comment:
```suggestion
Both of these options use [{{ecloud}}](/deploy-manage/deploy/elastic-cloud.md) as the orchestrator.
```

`as the orchestrator` better?
`as the orchestration platform`?

Whatever you prefer, `an orchestrator` just felt a bit weird but don't trust me much on this :)